### PR TITLE
Change test command for README documentation

### PR DESCRIPTION
**Pull request by louise-r-blue**
This PR changes the test command to use the correct rspec command in the README for open source contributors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ git clone <your fork>
 cp env-example .env
 bundle
 yarn
-bundle exec rails s
+bundle exec rspec spec
```
Comment (Rabid-Dan) [inline README.md]: You can probably just remove the spec bit of this. Running it as just Bundle exec rspec will run all of the specs anyway

**Comment by Rabid-Dan** [comment]
🐘 could this Pr have some kind of reference to the Readme in the title or the description (or both) please? Reading the title before I saw the changes made me think it was something completely different

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ git clone <your fork>
 cp env-example .env
 bundle
 yarn
-bundle exec rails s
+bundle exec rspec
```
Comment (vinithanatarajan) [inline README.md]: Above are the instructions to start server locally, may be leave `bundle exec rails s` here and add `bundle exec rspec` as well with some comments saying "run rspec tests".

Comment (louise-r-blue) [reply README.md]: @vinithanatarajan Yes my bad should be `bundle exec rspec` only in the testing section : }

**Review by elspeth-rabid — APPROVED** [review]
Nice!

File: README.md [hunk 3]
```diff
@@ -39,7 +39,6 @@ git clone <your fork>
 cp env-example .env
 bundle
 yarn
-bundle exec rails s
```
Comment (Rabid-Dan) [inline README.md:42]: I'm guessing this should probably have been left in?

**Review by Rabid-Dan — APPROVED** [review]

**Review by vinithanatarajan — APPROVED** [review]

**Review by ghostPepperWonton — APPROVED** [review]

**Pull request merged**